Documentation/specs/runtime-configuration-file.md

```diff
@@ -128,9 +122,13 @@ This section is copied verbatim from an identical section in the input `project.
 
 These settings are read by `corehost` to determine how to initialize the runtime. All versions of `corehost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
 
+### `targetRuntime` Section (`.runtimeconfig.json`)
+
+This section indicates the name and version of the shared runtime in use. The `dotnet publish` command will place this data here (where it gets that data from is undecided but out of the scope of this document). If not present, the host will assume the app is published as a standalone app. The presence of this section indicates if the application is standalone or not.
```

Comment:
Also parsing 2 files to get a sense of the app and one user editable doesn't sound like deps is self sufficient...